src/CGIHandler.cpp

```diff
@@ -1,8 +1,6 @@
 #include "CGIHandler.hpp"
-#include <vector>
-#include <sys/types.h>
-#include <sys/wait.h>
-#include <unistd.h>
+
+#define EXIT_FAILURE 1
```

Comment:
Can we add the proper header of EXIT_FAILURE instead of hard coding it?

Reply:
Done